PKGBUILD

```
@@ -0,0 +1,42 @@
+# Maintainer: Julian <juliandroske@users.noreply.github.com>
+
+pkgname=ethtool
+pkgver=6.11
+pkgrel=1
+pkgdesc="Utility for controlling network drivers and hardware"
+url="https://cdn.kernel.org/pub/software/network/ethtool"
+arch=(x86_64)
+license=(GPL-2.0-only)
+options=(!lto)
+depends=(
+  libmnl
+)
+makedepends=(
+  musl
+  linux-headers
+  libmnl
+)
+source=(
+  "https://cdn.kernel.org/pub/software/network/$pkgname/$pkgname-$pkgver.tar.gz"
+)
+sha256sums=('a8bee475d991939cad182b7dbcf3a1f139aeed8cb0add73d72084cd71d43282b')
+
+prepare() {
+  _patch_ ethtool-$pkgver
+}
+
+build() {
+  cd "$srcdir/$pkgname-$pkgver"
+  ./configure --prefix=/usr --mandir=/usr/share/man --sbindir=/usr/bin
```

nit: format it to ease reader's eyes:

```suggestion
  ./configure --prefix=/usr \
      --mandir=/usr/share/man  \
      --sbindir=/usr/bin
```